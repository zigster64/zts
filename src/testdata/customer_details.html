<div>
  <h1>Summary Stamement Page</h1>
  Details in this file, up to the 1st .section
  are included in the "printHeader()" function.

  .customer_details
  <h2>Customer</h2>
  <p>Name: {[name]s}</p>
  <p>Address: {[address]s}</p>
  <p>Credit Limit: $ {[credit]d:.2}</p>

  .invoice_table
  <h2>Invoices</h2>
  <table>
    <tr>
      <th>Date</th>
      <th>Details</th>
      <th>Amount</th>
    </tr>

    .invoice_row
    <tr>
      <td>{[date]s}</td>
      <td>{[details]s}</td>
      <td>$ {[amount]d:.2}</td>
    </tr>

    .invoice_total
    <tr>
      <td></td>
      <td>Total Due:</td>
      <td>$ {[total]d:.2}</td>
    </tr>

  </table>

  Note that there is no need to add a footer section -
  everything after the last .section is printed when
  that .section is printed.
</div>
