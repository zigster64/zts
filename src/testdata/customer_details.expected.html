<div>
  <h1>Summary Stamement Page</h1>
  Details in this file, up to the 1st .section
  are included in the "printHeader()" function.

  <h2>Customer</h2>
  <p>Name: Joe Blow</p>
  <p>Address: 21 Main Street</p>
  <p>Credit Limit: $ 100.00</p>

  <h2>Invoices</h2>
  <table>
    <tr>
      <th>Date</th>
      <th>Details</th>
      <th>Amount</th>
    </tr>

    <tr>
      <td>2023-10-01</td>
      <td>New Hoodie</td>
      <td>$ 80.99</td>
    </tr>

    <tr>
      <td>2023-10-03</td>
      <td>Hotdog with Sauce</td>
      <td>$ 4.50</td>
    </tr>

    <tr>
      <td>2023-10-04</td>
      <td>Mystery Gift</td>
      <td>$ 12.00</td>
    </tr>

    <tr>
      <td>2023-10-12</td>
      <td>Model Aircraft</td>
      <td>$ 48.00</td>
    </tr>

    <tr>
      <td>2023-10-24</td>
      <td>Chocolate Milkshake</td>
      <td>$ 80.99</td>
    </tr>

    <tr>
      <td></td>
      <td>Total Due:</td>
      <td>$ 226.48</td>
    </tr>

  </table>

  Note that there is no need to add a footer section -
  everything after the last .section is printed when
  that .section is printed.
</div>
